tmt/steps/report/reportportal.py

```diff
@@ -314,10 +314,14 @@ def go(self) -> None:
         attributes = [
             {'key': key, 'value': value[0]}
             for key, value in self.step.plan._fmf_context.items()]
-
         launch_attributes = self.construct_launch_attributes(suite_per_plan, attributes)
 
         launch_description = self.data.launch_description or self.step.plan.summary
+        # Check whether artifact URL has been provided
+        if not launch_description:
+            launch_description = os.getenv('TMT_REPORT_ARTIFACTS_URL')
```

Comment:
What about using a variable instead of repeating `os.getenv()` three times?

Reply:
Good, what about including the option as well?

Reply:
That is already the option, as `self.data.artifacts_url` is populated from fmf key, CLI option, or envvar.

Reply:
I meant 36a03b0887829ad8ecaded5c5d48a7b2e33038fd.